### PR TITLE
PROC-1295: Add missing shading to match old Maven implementation

### PR DESCRIPTION
**Pull request by zacharygoodwin**

### CODE REVIEW

File: proctor-ant-plugin/build.gradle [hunk 1]
```diff
@@ -1,3 +1,7 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '8.1.1'
```
Comment (vanhaxl) [inline proctor-ant-plugin/build.gradle:2]: Is this plugin to exclude dependencies that can cause expected issues?

Comment (vanhaxl) [reply proctor-ant-plugin/build.gradle:2]: I see people are also using in the build.gradle

        compile("com.indeed:proctor-common") {
            exclude group:'org.springframework'
            exclude group:'javax.servlet'
 Is this similar?

Comment (zacharygoodwin) [reply proctor-ant-plugin/build.gradle:2]: Shadow jar is not designed that greatly. Need to exclude other libraries or else we include them in our jar which can cause bugs down stream. I confirmed this by checking jar contents using `jar -tf path/to/X.jar `

Comment (vanhaxl) [reply proctor-ant-plugin/build.gradle:2]: Gotcha.

**Review by vanhaxl — APPROVED** [review]

**Review by stevenchen-indeed — APPROVED** [review]

**Pull request merged**